descriptions/ab+/spa.lua

```diff
@@ -741,103 +741,103 @@ EID.descriptions[languageCode].trinkets={
 	{"53", "Garrapata", "{{HealingRed}} Cura 1 corazón rojo al entrar a una {{BossRoom}} Sala de Jefe#El jefe comienza con -15% de vida#{{Warning}} Al recogerlo, no puede ser soltado#Solo es removido con el {{Trinket41}} Fósforo o tragándolo"}, -- Tick
 	{"54", "Cabeza de Isaac", "Familiar con lágrimas penetrantes#Inflige 3.5 de daño"}, -- Isaac's Head
 	{"55", "La fe de Maggy", "{{EternalHeart}} Otorga un Corazón Eterno al entrar a una nueva planta"}, -- Maggy's Faith
 	{"56", "Lengua de Judas", "{{DevilRoom}} Reduce los precios de los pactos del Diablo a un corazón#{{SoulHeart}} No reduce los pactos de 3 Corazones de Alma"}, -- Judas' Tongue
-	{"57", "Alma de ???", "Familiar que rebota por la habitación#Dispara en la misma dirección que el jugador#Inflige 3.5 de daño"}, -- ???'s Soul
-	{"58", "Mechón de Sansón", "[[Timer}} Matar a un enemigo tiene 1/15 de probabilidad de otorgar ↑ {{Damage}} Daño +0.5 por la habitación#{{Luck}} 100% de probabilidad con 10 de suerte"}, -- Samson's Lock
+	{"57", "Alma de ???", "Familiar que rebota por la habitación#Dispara en la misma dirección que el jugador#Inflige 3.5 de daño#Tiene lágrimas espectrales y teledirigidas"}, -- ???'s Soul
+	{"58", "Mechón de Sansón", "{{Timer}} Matar a un enemigo tiene 1/15 de probabilidad de otorgar ↑ {{Damage}} Daño +0.5 por la habitación#{{Luck}} 100% de probabilidad con 10 de suerte"}, -- Samson's Lock
 	{"59", "Ojo de Caín", "{{Collectible21}} 25% de probabilidad de revelar los iconos del mapa al empezar un nuevo piso#{{Luck}} 100% de probabilidad con 3 de suerte"}, -- Cain's Eye
 	{"60", "Pata del Ave de Eva", "{{Collectible117}} 5% de probabilidad de generar un pájaro muerto al matar un enemigo#{{Luck}} 100% de probabilidad con 8 de suerte"}, -- Eve's Bird Foot
 	{"61", "La Mano izquierda", "{{RedChest}} Reemplaza todos los cofres por cofres rojos"}, -- La Mano izquierda
 	{"62", "Rocas brillantes", "Las rocas marcadas y de trampilla brillan cada 10 segundos"}, -- Shiny Rock
 	{"63", "Tijeras de Seguridad", "{{Bomb}} Convierte las bombas troll en bombas recolectables"}, -- Safety Scissors
 	{"64", "Gusano de Arcoíris", "Otorga un efecto de gusano aleatorio cada 3 segundos"}, -- Rainbow Worm
-	{"65", "Gusano Estirado", "↑ {{Range}} Alcance x2#↓ {{Tearsize}} Tamaño de lágrima x0.5"}, -- Tape Worm
-	{"66", "Gusano Perezoso", "↑ {{Range}} Alcance +4#↑ {{Tearsize}} Tamaño de lágrima +2#↓ {{Shotspeed}} Vel. de lágrimas -0.4#"}, -- Lazy Worm
-	{"67", "Dado Roto", "Recibir daño tiene un 50% de probabilidad de activar uno de los siguientes efectos:#{{Collectible105}} D6#{{Collectible406}} D8#{{Collectible386}} D12 #{{Collectible166}} D20"},
+	{"65", "Gusano Estirado", "↑ {{Range}} Alcance x2#↓ Altura de lágrima x0.5"}, -- Tape Worm
+	{"66", "Gusano Perezoso", "↑ {{Range}} Alcance +4#↑ Altura de lágrima +2#↓ {{Shotspeed}} Vel. de lágrimas -0.5#"}, -- Lazy Worm
```

Comment:
minor correction for the old file
```suggestion
	{"66", "Gusano Perezoso", "↑ {{Range}} Alcance +4#↑ Altura de lágrima +2#↓ {{Shotspeed}} Vel. de lágrimas -0.5"}, -- Lazy Worm
```
good